src/Microsoft.PowerShell.Commands.Management/commands/management/Hotfix.cs

```diff
@@ -90,7 +75,15 @@ protected override void BeginProcessing()
             {
                 bool foundRecord = false;
                 StringBuilder QueryString = new StringBuilder();
-                ConnectionOptions conOptions = ComputerWMIHelper.GetConnectionOptions(AuthenticationLevel.Packet, ImpersonationLevel.Impersonate, this.Credential);
+                ConnectionOptions conOptions = new ConnectionOptions();
+                conOptions.Authentication = AuthenticationLevel.Packet;
+                conOptions.Impersonation = ImpersonationLevel.Impersonate;
+                if (Credential != null)
+                {
+                    conOptions.Username = Credential.UserName;
+                    conOptions.SecurePassword = Credential.Password;
+                }
```

Comment:
We could use an initializer:
```suggestion
                ConnectionOptions conOptions = new ConnectionOptions()
                {
                    ....
                    conOptions.Username = Credential?.UserName;
                }
```

Reply:
will update